client/src/pages/login.js

```diff
@@ -21,14 +20,14 @@ const Index = (props) => {
         </Heading>
       </Flex>
       <Flex>
-        <ChakraLink m={4} href={`${props.SERVER_URL}/spotify/login`}>
-          <Button as={ChakraLink}>Login</Button>
+        <ChakraLink m={4} href={process.env.NEXT_PUBLIC_AUTH_URL}>
```

Comment:
Then, this would be `` `${process.env.NEXT_PUBLIC_SERVER_URL}/spotify/login` ``.  
Otherwise, it may be confusing for the team to set up so many variables.

Reply:
Not quite, the auth url is not even in our server domain, it's `https://accounts.spotify.com/authorize?client_id=<client-id-here>&response_type=code&redirect_uri=http://localhost:9002/spotify/login&scope=streaming%20user-read-email%20user-read-private%20user-library-read%20user-library-modify%20user-read-playback-state%20user-modify-playback-state%20playlist-read-private`. 

Initially, we had a lot of useless template code just to wrap the auth_url. This makes things simplier. Also, it's just one variable :P.